vertx-web-openapi/src/main/java/io/vertx/ext/web/openapi/impl/OpenAPIHolderImpl.java

```diff
@@ -43,21 +45,8 @@
   private final YAMLMapper yamlMapper;
   private JsonObject openapiRoot;
 
-  private static URI openapiSchemaURI;
-  private static JsonObject openapiSchemaJson;
-
-  static {
-    try {
-      openapiSchemaURI = OpenAPIHolderImpl.class.getResource("/openapi_3_schema.json").toURI();
-      openapiSchemaJson = new JsonObject(
-          String.join("",
-              Files.readAllLines(Paths.get(openapiSchemaURI))
-          )
-      );
-    } catch (Exception e) {
-      throw new RuntimeException(e);
-    }
-  }
+  // Hardcoded FTW
```

Comment:
can you make it multiline ?

also can you compute the hashcode and in a test check the hashcode so we don't have modifications ?

Reply:
> can you make it multiline ?

Why? it's 1654 lines beautified, multiline is bigger in terms of size and noisy 

> also can you compute the hashcode and in a test check the hashcode so we don't have modifications ?

We could potentially update this schema in future releases

Reply:
Now this should be better